src/components/marquee-wizard/marquee-wizard.component.html

```diff
@@ -47,7 +45,13 @@
                 [attr.aria-setsize]="steps.length"
                 [attr.aria-selected]="step.active"
                 [attr.aria-controls]="id + '-step-' + index"
+                [attr.aria-labelledby]="id + '-step-' + index + '-label'"
```

Comment:
Some of the properties were being created in the component and most in the template. As there is no real requirement to create the likes of IDs in the component I removed the injected dependencies and added them here.